routes/cube_routes.js

```diff
@@ -545,114 +545,92 @@ router.get('/blogsrc/:id', function(req, res) {
-                pages: pages,
-                metadata: generateMeta(
-                  `Cube Cobra Blog: ${cube.name}`,
-                  (cube.type) ? `${cube.card_count} Card ${cube.type} Cube` : `${cube.card_count} Card Cube`,
-                  cube.image_uri,
-                  `https://cubecobra.com/cube/blog/${req.params.id}`
-                ),
-                loginCallback: '/cube/blog/' + req.params.id
-              });
-            } else {
-              res.render('cube/cube_blog', {
-                cube: cube,
-                cube_id: cube_id,
-                owner: user.username,
-                activeLink: 'blog',
-                title: `${abbreviate(cube.name)} - Blog`,
-                posts: blogs,
-                metadata: generateMeta(
-                  `Cube Cobra Blog: ${cube.name}`,
-                  (cube.type) ? `${cube.card_count} Card ${cube.type} Cube` : `${cube.card_count} Card Cube`,
-                  cube.image_uri,
-                  `https://cubecobra.com/cube/blog/${req.params.id}`
-                ),
-                loginCallback: '/cube/blog/' + req.params.id
-              });
-            }
-          } else {
-            res.render('cube/cube_blog', {
-              cube: cube,
-              cube_id: cube_id,
-              owner: user.username,
-              activeLink: 'blog',
-              title: `${abbreviate(cube.name)} - Blog`,
-              metadata: generateMeta(
-                `Cube Cobra Blog: ${cube.name}`,
-                (cube.type) ? `${cube.card_count} Card ${cube.type} Cube` : `${cube.card_count} Card Cube`,
-                cube.image_uri,
-                `https://cubecobra.com/cube/blog/${req.params.id}`
-              ),
-              loginCallback: '/cube/blog/' + req.params.id
-            });
-          }
-        });
-      });
+        } else {
+          pages.push({
+            url: '/cube/blog/' + cube_id + '/' + i,
+            content: (i + 1)
+          });
+        }
+      }
+      blog_page = [];
+      for (var i = 0; i < 10; i++) {
+        if (blogs[i + page * 10]) {
+          blog_page.push(blogs[i + page * 10]);
+        }
+      }
     }
-  });
+
+    return res.render('cube/cube_blog', {
+      cube: cube,
+      cube_id: cube_id,
+      owner: user.username,
+      activeLink: 'blog',
+      title: `${abbreviate(cube.name)} - Blog`,
+      posts: blogs.length > 0 ? blog_page : blogs,
+      pages: blogs.length > 0 ? pages : null,
+      metadata: generateMeta(
+        `Cube Cobra Blog: ${cube.name}`,
+        (cube.type) ? `${cube.card_count} Card ${cube.type} Cube` : `${cube.card_count} Card Cube`,
+        cube.image_uri,
+        `https://cubecobra.com/cube/blog/${req.params.id}`
+      ),
+      loginCallback: '/cube/blog/' + req.params.id
+    });
+  } catch (err) {
+    return res.status(500).send(err);
+  }
 });
 
-router.get('/blog/:id/rss', function(req, res) {
+router.get('/rss/:id', function(req, res) {
```

Comment:
what's the motivation for the url change here? I don't think i minded `/blog/:id/rss`...

Reply:
there might be people where this is a breaking change?